shared/controller-api/src/main/java/io/pravega/shared/controller/event/kvtable/CreateTableEvent.java

```diff
@@ -68,7 +70,8 @@ protected byte getWriteVersion() {
 
         @Override
         protected void declareVersions() {
-            version(0).revision(0, this::write00, this::read00);
+            version(0).revision(0, this::write00, this::read00)
```

Comment:
Do we need to do this? Is `CreateTableEvent` used for anything other than KVTs? If not, then we may not need to worry about backwards compatibility (in this particular scenario). Please double check before taking my suggestion for granted :)

Reply:
As with the previous case, since this change is not backwards compatible anyways I think there is little reason to have an new revision. The new serialization is being done in the original revision.